node/index.js

```diff
@@ -437,64 +255,8 @@ const prettyPrintResponse = (response) => {
   console.log(util.inspect(response, { colors: true, depth: 4 }));
 };
 
-// This is a helper function to poll for the completion of an Asset Report and
-// then send it in the response to the client. Alternatively, you can provide a
-// webhook in the `options` object in your `/asset_report/create` request to be
-// notified when the Asset Report is finished being generated.
-const respondWithAssetReport = (
-  numRetriesRemaining,
-  assetReportToken,
-  client,
-  response,
-) => {
-  if (numRetriesRemaining == 0) {
-    return response.json({
-      error: 'Timed out when polling for Asset Report',
-    });
-  }
-
-  const includeInsights = false;
-  client.getAssetReport(
-    assetReportToken,
-    includeInsights,
-    function (error, assetReportGetResponse) {
-      if (error != null) {
-        prettyPrintResponse(error);
-        if (error.error_code == 'PRODUCT_NOT_READY') {
-          setTimeout(
-            () =>
-              respondWithAssetReport(
-                --numRetriesRemaining,
-                assetReportToken,
-                client,
-                response,
-              ),
-            1000,
-          );
-          return;
-        }
-
-        return response.json({
-          error,
-        });
-      }
-
-      client.getAssetReportPdf(
-        assetReportToken,
-        function (error, assetReportGetPdfResponse) {
-          if (error != null) {
-            return response.json({
-              error,
-            });
-          }
-
-          response.json({
-            error: null,
-            json: assetReportGetResponse.report,
-            pdf: assetReportGetPdfResponse.buffer.toString('base64'),
-          });
-        },
-      );
-    },
-  );
+const makeErrorObject = (error) => {
+  return {
+    error: { ...error.data, status_code: error.status },
+  };
```

Comment:
we have helper functions in at least ruby and python that do this same thing and they are called `formatError`. We could use the same name for this function for consistency!

Reply:
We rm'd these helper functions. They are not generated.

Reply:
i renamed.